package.json

```diff
@@ -43,11 +43,11 @@
     "esvalid": "1.1.0",
     "fs-readdir-recursive": "0.1.0",
     "jshint": "2.5.11",
-    "lodash": "2.4.1",
+    "lodash": "3.0.0",
```

Comment:
@jdalton Yeah, I used to have them all as carets but then a package broke in a patch and I got angry and made all the versions fixed. It was supposed to be a stop-gap solution but I just never got around to changing them back.